frontend/src/components/Collections.vue

```diff
@@ -15,7 +15,7 @@
     <v-row>
       <v-col cols="12">
         <div v-for="(collection, index) in collections" :key="index"><br>
-          <v-card-title class="prompt-title" style="font-size:16px">{{collection.title}}</v-card-title>
+          <v-card-title class="prompt-title" style="margin-bottom:8px;font-size:20px">{{collection.title}}</v-card-title>
```

Comment:
why inline style? 
we have the div.prompt-title class in globalStyles.css

Reply:
done